projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -138,7 +138,12 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
     }
 
     protected get hasSelectedItem(): boolean {
-        return !!this.selectionService.get(this.id).size;
+        if (this.comboInput.value === '') {
```

Comment:
The problem is that because the simple combo can accept `falsy` values, during initialization of the `FormControl`, such a value is inserted in the selection service. IMO instead of filtering these out, we should be preventing the selection service from receiving such `falsy` values that are not considered valid by the `FormControl`'s required validator. This should only apply if the component is part of a reactive form or has `ngModel`; otherwise it should allow any `falsy` value to be added to the selection service with the exception of `undefined` since iirc we use it to signify that no value is selected.